apps/pigment-css-next-app/package.json

```diff
@@ -10,14 +10,8 @@
   },
   "dependencies": {
     "@pigment-css/react": "workspace:^",
-    "@mui/utils": "next",
-    "@mui/base": "next",
-    "@mui/lab": "next",
-    "@mui/material": "next",
-    "@mui/system": "next",
-    "@mui/material-nextjs": "next",
-    "@mui/icons-material": "next",
-    "@emotion/cache": "latest",
+    "@mui/material": "^5.16.6",
+    "@mui/material-nextjs": "^5.16.6",
```

Comment:
Can you change to `6.0.0-beta.2` if the latest beta fails.

Reply:
It does not make sense to use v5.

Reply:
beta.2 also has the same issue.